scripts/preprocess/process_dictionaries.R

```diff
@@ -0,0 +1,51 @@
+process_state_dictionary <- function(outfile, county_boundaries_census_sp) {
+  # get unique FIPS both from the sp data and dataRetrieval
+  unique_fips <- sort(unique(as.character(county_boundaries_census_sp@data$STATEFP)))
+  dr_states <- dataRetrieval::stateCd %>%
+    transmute(STATE_FIPS=STATE, STATE_ABBV=STUSAB, STATE_NAME=STATE_NAME)
+  
+  # check for unrecognized FIPS codes
+  bad_fips <- unique_fips[!(unique_fips %in% dr_states$STATE_FIPS)]
+  if(length(bad_fips) > 0) {
+    stop('these state FIPS are unknown to dataRetrieval: ', paste(bad_fips, collapse=', '))
```

Comment:
It might be useful to state that we are using `dataRetrieval` as the authoritative source for county and state fips in the yaml file rather than buried down here.

Reply:
i don't really think we are deferring to dataRetrieval for the FIPS codes; we're just checking for inconsistencies and making sure (by this check) that we'd know if there were any. we're just adding in STATE_ABBV and STATE_NAME using dataRetreival. i don't see that addition as rising to the level of needing a comment in the yaml